CONTRIBUTING.md

```diff
@@ -30,103 +73,115 @@ $ git clone https://github.com/exercism/interactive-cli-walkthrough
 $ bundle
 ```
 
-## Submitting Code Changes
+### Writing workflow
 
-These instructions should get you closer to getting a commit into the
-repository.
+#### Files
 
-### Git Workflow
+##### Format
 
-1. Fork and clone.
-1. Add the upstream interactive-cli-walkthrough repository as a new remote to your clone.
-   `git remote add upstream https://github.com/exercism/interactive-cli-walkthrough.git`
-1. Create a new branch
-   `git checkout -b name-of-branch`
-1. Commit and push as usual on your branch.
-1. When you're ready to submit a pull request, rebase your branch onto
-   the upstream master so that you can resolve any conflicts:
-   `git fetch upstream && git rebase upstream/master`
-   You may need to push with `--force` up to your branch after resolving conflicts.
-1. When you've got everything solved, push up to your branch and send the pull request as usual.
+Files are written in Markdown. If you haven't used it before, we recommend
+looking at this [tutorial][markdown].
 
-### Issues
+Files should be saved with the `*.tw2` extension.
 
-We keep track of bugs, enhancements and support requests in the repository using GitHub [issues][].
+##### Including files
 
-### Pull Requests
+The main file which includes everything is [main.tw2][main.tw2]. In order for
+your file to appear in the walkthrough, [main.tw2][main.tw2] should contain the
+path to your file.
```

Comment:
I found myself feeling a bit lost in this section. I was thinking "what does include everything mean?" and "what would the path be?".

I wonder if it would be useful to have a quick overview of the "architecture" of the walkthrough. E.g. We support 3 operating systems. Some instructions are shared between all the operating systems, some are specific to just one. Each step goes in a separate file. All the files are listed in the `main.tw2` file, in the order that the steps would appear. Each step consists of X and Y, which should have something-something particular format.

Or something like that. I might be misunderstanding how the walkthrough is put together :)